$OpenBSD: patch-Makefile,v 1.2 2020/06/08 21:07:17 edd Exp $

Don't compresss manuals and remove odd symlinking scheme to alias man pages. As
it happens mandoc is clever enough to figure out the aliases.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -122,12 +122,9 @@ install-man:
 	install -d $(DESTDIR)$(MAN3DIR)
 	install -d $(DESTDIR)$(MAN7DIR)
 	for F in man/*.3; do \
-	  gzip <$$F >$(DESTDIR)$(MAN3DIR)/$${F#man/}.gz; \
+	  install -c $$F $(DESTDIR)$(MAN3DIR)/$${F#man/}; \
 	done
 	for F in man/*.7; do \
-	  gzip <$$F >$(DESTDIR)$(MAN7DIR)/$${F#man/}.gz; \
+	  install -c $$F $(DESTDIR)$(MAN7DIR)/$${F#man/}; \
 	done
-	while read FROM EQ TO; do \
-	  echo ln -sf $$TO.gz $(DESTDIR)$(MAN3DIR)/$$FROM.gz; \
-	done < man/also
 
